app/move/views/_includes/assessment.njk

```diff
@@ -92,7 +92,7 @@
         }) }}
       {% endif %}
     {% endif %}
-  {% endif %}
 
-  {{ updateLink(updateLinks.court) }}
+    {{ updateLink(updateLinks.court) }}
+  {% endif %}
 </section>
```

Comment:
I think it might be better to move this whole `section` within that block. Otherwise in some instances we'd be left with just an empty `<section>` tab on the page.

Reply:
👍